### ISSUE TITLE
Floating Point Unit is instantiated in CORE-V-MCU

### ISSUE THREAD

**Issue by MikeOpenHWGroup** [openhwgroup/core-v-mcu#190]
The CORE-V-MCU instantiates the "fpnew" floating point IP from the pulp-platform team.  As discussed in #189, the CV32E40P does not make use of this floating point unit, so it really should not be there.

Note: I have labelled this as a bug for two reasons:
1. Instantiating fpnew and connecting it to the core via the APU bus may cause unintended behaviour.  This has not been tested for.
2. Physical Implementation: even if the instantiating and connecting fpnew does not cause a functional bug, it is not good practise to include a lot of unused logic.

**Comment by MikeOpenHWGroup** [comment]
Fixed in #191

**Comment by gmartin102** [comment]
closed -- FPU is removed from cv32e40p instantiation

**Comment by MikeOpenHWGroup** [comment]
> FPU is removed from cv32e40p instantiation

As of `53a5d08`, the FPU is still there.

The parameter is not set to 0, so the CV32E40P will no longer decode F or D instructions to be forwarded to the APU.  This is good.  However the floating point IP is still instantiated in the MCU and is still connected to the CV32E40P core via the APU interface.

Is that what we want?

**Comment by MikeOpenHWGroup** [comment]
Yes, you are right that the instantiation of cv32e40 indicates `USE_FPU=0`.  This means that the core will now flag floating point instructions as illegal instructions.

However, the FPU IP is still instantiated and connected to the core in the [fc_subsystem](https://github.com/openhwgroup/core-v-mcu/blob/master/rtl/core-v-mcu/fc/fc_subsystem.sv#L214).  The core can no longer use it, but the floating point unit instance still exists in the RTL model and will still be synthesized into gates.
